scripts/jxrenc_0.py

```diff
@@ -84,6 +91,7 @@ def _determine_q(max_q, max_file_size_kb, output_path, tif_path):
         prev_q = q
 
 def encode_jxr(printProgress=False, maxFileSizeKb = 32):
+    i = 0
     number_of_files = len(glob.glob('Images/' + '*/' + '*' + pngExtension))
```

Comment:
you need to adapt number_of_files like in the others and remove in the also the commented out code